src/tests/test_dal.py

```diff
@@ -0,0 +1,32 @@
+from polydatum import DataAccessLayer, DataManager as _BaseDataManager
+
+
+def test_custom_dal():
+    """
+    Verify that a custom DataAccessLayer can be passed to DataManager
+    """
+
+    class TestDal(object):
+        """
+        Test dal class
+        """
+        def __init__(self, data_manager):
+            self._services = {}
+            self._data_manager = data_manager
+
+    class TestDataManager(_BaseDataManager):
+        """
+        DataManager wrapper for test
+        """
+        DataAccessLayer = TestDal
+
+        def __init__(self, resource_manager=None):
```

Comment:
This override is not needed. Putting it in the test makes it seem significant and adds cognitive overload to the test.